### PR TITLE
Convert to Swift 3.0

### PR DESCRIPTION
**Pull request by minhchau273**
@chug2k @harley Check it out. If everything looks good, please merge this PR before class tomorrow. 💃

### CODE REVIEW

File: Yelp/Business.swift [hunk 1]
```diff
@@ -84,11 +84,11 @@ class Business: NSObject {
         return businesses
     }
     
-    class func searchWithTerm(term: String, completion: ([Business]!, NSError!) -> Void) {
-        YelpClient.sharedInstance.searchWithTerm(term, completion: completion)
+    class func searchWithTerm(_ term: String, completion: @escaping ([Business]?, Error?) -> Void) {
```
Comment (chug2k) [inline Yelp/Business.swift:87]: if we want to be very Swift 3.0....the signature would be something like `class func search(with term: String, completion...`

File: Yelp/YelpClient.swift [hunk 2]
```diff
@@ -18,23 +18,19 @@ let yelpToken = "uRcRswHFYa1VkDrGV6LAW2F8clGh5JHV"
...
-    class var sharedInstance : YelpClient {
-        struct Static {
-            static var token : dispatch_once_t = 0
-            static var instance : YelpClient? = nil
-        }
-        
-        dispatch_once(&Static.token) {
-            Static.instance = YelpClient(consumerKey: yelpConsumerKey, consumerSecret: yelpConsumerSecret, accessToken: yelpToken, accessSecret: yelpTokenSecret)
+
+    static var _sharedInstance: YelpClient?
```
Comment (chug2k) [inline Yelp/YelpClient.swift]: Swift 3.0 seems to prefer to call this _shared now, and shared() below

**Review by chug2k — APPROVED** [review]
wonderful job. i thought i'd just say a few things based on what i've learned very recently about swift 3.0

**Comment by harley** [comment]
awesome Chau. hehe you're in good hands. i'll just watch 🍿

File: Yelp/Business.swift (hunk 1 above)
Comment (minhchau273) [reply Yelp/Business.swift:87]: Agree! 😄

File: Yelp/Business.swift [hunk 3]
```diff
@@ -85,10 +85,10 @@ class Business: NSObject {
     }
     
     class func searchWithTerm(_ term: String, completion: @escaping ([Business]?, Error?) -> Void) {
```
Comment (chug2k) [inline Yelp/Business.swift:87]: this one could also be refactored 😛  but THIS IS ALREADY GREAT. :) @harley I'll let you merge

**Pull request merged**